.github/workflows/bare.yaml

```diff
@@ -14,19 +14,11 @@ concurrency:
   group: bare-${{ github.ref }}
   cancel-in-progress: true
 
-# Set to 'yes' to open a tunnel to GitHub's VMs through ngrok/tmate on failures.
-# Also increase timeout-minutes for the relevant OS when debugging remotely.
-# Version 3 of mxschmitt/action-tmate should also work on Windows.
-env:
-  TMATE_DEBUG: 'no'
-  NGROK_DEBUG: 'no'
-
-
 jobs:
   windows:
     # The type of runner that the job will run on
     runs-on: windows-latest
-    timeout-minutes: 15
+    timeout-minutes: ${{ matrix.tests.debug == 'yes' && 45 || 15 }}
```

Comment:
do we have a matrix here ?

Reply:
What do you suggest? I've tried using `runner.debug` but it wouldn't work: _Unrecognized named-value: 'runner'_.

I've also tried simply `test.debug`, to no avail.

Reply:
The `runner` is not available at the "jobs->JOB_ID" level.
It is only avalable at the `jobs->JOB_ID->steps->STEP_ID` level

Just leave a big timeout here, and make sure each step has a lower timeout, with the exception of the debug step

Reply:
Fixed in f9d6da61930cc40e6f555359cb3e02ead8fe7130.